hazelcast/src/test/java/com/hazelcast/internal/util/JavaVersionTest.java

```diff
@@ -0,0 +1,77 @@
+package com.hazelcast.internal.util;
+
+import com.hazelcast.test.HazelcastSerialClassRunner;
+import com.hazelcast.test.HazelcastTestSupport;
+import com.hazelcast.test.annotation.ParallelTest;
+import com.hazelcast.test.annotation.QuickTest;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.runner.RunWith;
+
+
+import static com.hazelcast.internal.util.JavaVersion.JAVA_1_6;
+import static com.hazelcast.internal.util.JavaVersion.JAVA_1_7;
+import static com.hazelcast.internal.util.JavaVersion.JAVA_1_8;
+import static com.hazelcast.internal.util.JavaVersion.JAVA_1_9;
+import static com.hazelcast.internal.util.JavaVersion.UNKNOWN;
+import static org.junit.Assert.*;
+
+@RunWith(HazelcastSerialClassRunner.class)
+@Category({QuickTest.class, ParallelTest.class})
+public class JavaVersionTest extends HazelcastTestSupport {
+
+    @Test
+    public void parseVersion() throws NoSuchFieldException, IllegalAccessException {
```

Comment:
You can simplify this to `throws Exception`, it doesn't matter for the test and is less to read.